nebula/ui/components/VPNServerList.qml

```diff
@@ -62,77 +129,306 @@ FocusScope {
+
+                    onClicked: {
+                        focusScope.updateServerData();
+                    }
+
+                    RowLayout {
+                        anchors {
+                            fill: parent
+                            leftMargin: VPNTheme.theme.listSpacing
+                            rightMargin: VPNTheme.theme.listSpacing
+                        }
+                        spacing: VPNTheme.theme.listSpacing
+
+                        VPNIcon {
+                            source: "qrc:/nebula/resources/clock.svg"
+
+                            Layout.alignment: Qt.AlignLeft | Qt.AlignTop
+                            Layout.topMargin: VPNTheme.theme.listSpacing
+                        }
+
+                        VPNInterLabel {
+                            id: statusTitle
+
+                            Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
+                            Layout.fillWidth: true
+                            Layout.preferredWidth: parent.width
+                            Layout.maximumWidth: parent.width
+
+                            color: VPNTheme.theme.fontColor
+                            horizontalAlignment: Text.AlignLeft
+                            // TODO: Replace string arg with the actual formatted date
+                            text: !statusComponent.rowShouldBeDisabled
+                                ? VPNl18n.ServersViewRecommendedTextStatus.arg("5m")
+                                : VPNl18n.ServersViewRecommendedTextStatusWithInfo.arg("5m")
+                            wrapMode: Text.WordWrap
+                        }
+
+                        Item {
+                            Accessible.name: VPNl18n.ServersViewRecommendedTextInfo
+                            Layout.alignment: Qt.AlignRight | Qt.AlignVCenter
+
+                            height: VPNTheme.theme.iconSize * 1.5
+                            visible: !statusComponent.rowShouldBeDisabled
+                            width: VPNTheme.theme.iconSize * 1.5
+
+                            VPNIcon {
+                                id: refreshIcon
+                                source: "qrc:/nebula/resources/refresh.svg"
+                                sourceSize {
+                                    height: parent.height
+                                    width: parent.width
+                                }
+                            }
+
+                            VPNColorOverlay {
+                                anchors.fill: parent
+
+                                color: VPNTheme.colors.blue
+                                source: refreshIcon
+                            }
+                        }
+                    }
+                }
+
+                Repeater {
+                    id: recommendedRepeater
+                    model: testRecommendedModel
+                    delegate: VPNClickableRow {
```

Comment:
The recommended servers need accessible names

Reply:
Yes, it does. Added!